res/dark/launch-icon.svg

```diff
@@ -0,0 +1,3 @@
+<svg width="24" height="24" viewBox="0 0 24 24" fill="none" xmlns="http://www.w3.org/2000/svg">
```

Comment:
This looks like it should be the debug-icon (to match cpptools).
Launch should be just the triangle with no bug on it.